go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
```

Comment:
Review the compatibility versioning used in your dependencies (e.g., `+incompatible`). If possible, work towards using compatible versions to ensure robust dependency management.